src/test/java/org/commcare/formplayer/tests/BaseTestClass.java

```diff
@@ -690,6 +712,33 @@ CommandListResponseBean doUpdate(String requestPath) throws Exception {
         FORM, MENU, UTIL, DEBUGGER,
     }
 
+    private <T> T mockInstallReference(CheckedSupplier<T> supplier, String installReference) throws Exception {
```

Comment:
This isn't ideal but a better solution will require making more changes to the production code, particularly the `FormplayerConfigEngine`. I'll make a note to tackle that separately.